packages/inpage/setup-inpage-provider.js

```diff
@@ -20,42 +20,9 @@ function validateMessage(e) {
 }
 
 const FLUENT_BACKEND_PREFERENCES = '__FLUENT_BACKEND_PREFERENCES__'
-const FLUENT_USE_MODERN_PROVIDER_API = '__FLUENT_USE_MODERN_PROVIDER_API__'
-const FLUENT_OVERRIDE_WINDOW_DOT_CONFLUX =
-  '__FLUENT_OVERRIDE_WINDOW_DOT_CONFLUX__'
 const FLUENT_OVERRIDE_WINDOW_DOT_ETHEREUM =
   '__FLUENT_OVERRIDE_WINDOW_DOT_ETHEREUM__'
 
-function maybeUseModernProviderAPI(useModernProviderAPI) {
-  if (useModernProviderAPI) {
-    if (window.localStorage.getItem(FLUENT_USE_MODERN_PROVIDER_API)) return
-    window.localStorage.setItem(FLUENT_USE_MODERN_PROVIDER_API, true)
-    window.location.reload()
-  } else {
-    if (!window.localStorage.getItem(FLUENT_USE_MODERN_PROVIDER_API)) return
-    window.localStorage.removeItem(FLUENT_USE_MODERN_PROVIDER_API)
-    window.location.reload()
-  }
-}
-
-function maybeOverrideWindowDotConflux(overrideWindowDotConflux) {
-  if (overrideWindowDotConflux) {
-    if (
-      window.localStorage.getItem(FLUENT_OVERRIDE_WINDOW_DOT_CONFLUX) === false
-    ) {
-      window.localStorage.removeItem(FLUENT_OVERRIDE_WINDOW_DOT_CONFLUX)
-      window.location.reload()
-    }
-  } else {
-    if (
-      window.localStorage.getItem(FLUENT_OVERRIDE_WINDOW_DOT_CONFLUX) === false
-    )
-      return
-    window.localStorage.setItem(FLUENT_OVERRIDE_WINDOW_DOT_CONFLUX, false)
-    window.location.reload()
-  }
-}
-
 function maybeOverrideWindowDotEthereum(overrideWindowDotEthereum) {
   if (overrideWindowDotEthereum) {
     if (!window.localStorage.getItem(FLUENT_OVERRIDE_WINDOW_DOT_ETHEREUM)) {
```

Comment:
Both key and value of localstorage are strings. Better use JSON.parse. (JSON.parse will throws a [SyntaxError] exception if the string to parse is not valid JSON. )
 such as  try { JSON.pars(...)} catch (e) {...}

Reply:
There's no difference here since we only check the existance of the item. We never set it to false

Reply:
OK.That make sense

Reply:
I changed `true` to `'true'` here to emphasize the behavior